packages/front/src/store/modal.tsx

```diff
@@ -0,0 +1,18 @@
+import { ModalStore } from "@/interfaces";
+import { create } from "zustand";
+
+export const useModal = create<ModalStore>((set, get) => ({
```

Comment:
@alexandrearaujoo this store only used on confirm deposit? u can implement this in withdraw modal to?